### PR TITLE
Fix regex

### PR DESCRIPTION
**Pull request by germainelee02**

### CODE REVIEW

**Review by bernicetoh — COMMENTED** [review]
Great thanks!

File: src/main/java/seedu/recipe/model/recipe/Description.java [hunk 1]
```diff
@@ -10,9 +10,10 @@
 public class Description {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "A description should only contain alphanumeric characters and spaces, and it should not be blank";
+            "A description should only contain alphanumeric characters, "
+                    + "spaces, and these punctuation marks `,.!?`, and it should not be blank";
 
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum},.!?][\\p{Alnum} ,.!?]*";
```
Comment (kyueran) [inline src/main/java/seedu/recipe/model/recipe/Description.java:16]: Great!

File: src/main/java/seedu/recipe/model/recipe/Step.java [hunk 2]
```diff
@@ -10,9 +10,10 @@
 public class Step {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "A step should only contain alphanumeric characters and spaces, and it should not be blank";
+            "A step should only contain alphanumeric characters, "
+                    + "spaces, and these punctuation marks `,.!?`, and it should not be blank";
 
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum},.!?][\\p{Alnum} ,.!?]*";
```
Comment (kyueran) [inline src/main/java/seedu/recipe/model/recipe/Step.java:16]: lgtm

**Review by kyueran — COMMENTED** [review]
lgtm!

**Pull request merged**